shell/app/common/components/card-list/index.tsx

```diff
@@ -129,7 +129,7 @@ const CardList = <T extends unknown>({
                   rowId = record[rowKey];
                 }
                 const rowClass = classnames(
-                  'shadow-card mb-4 mx-2 px-4 rounded-sm transition-all duration-300 hover:bg-grey',
+                  'border-0 border-b border-solid border-default-08 mx-2 px-4 rounded-sm transition-all duration-300 hover:bg-grey',
```

Comment:
Why use `border-0` with `border-xx` ?

Reply:
if without 'border-0' before 'border-b'
![image](https://user-images.githubusercontent.com/30014895/152962099-0f1c45f5-8cf8-4e21-84db-97e8e13c4d9d.png)

the style will be displayed like this:
![image](https://user-images.githubusercontent.com/30014895/152962189-d96e08f1-e75e-49f9-8e25-0e8a1db8b5c9.png)

Some file styles in the project are also written like this
![image](https://user-images.githubusercontent.com/30014895/152962536-ca913528-8b93-42d0-b5cd-a0bc1bbc5542.png)